### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by gabrielsroka**
fixed a few minor typos/errors. i'm not positive about the parens on Q. 54

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1298,7 +1298,7 @@ A string is an iterable. The spread operator maps every character of an iterable
 
 ---
 
-###### 44. What's the output?
+###### <a name="20190621"></a>44. What's the output?
```
Comment (lydiahallie) [inline README.md]: Why did you add this?

Comment (lydiahallie) [reply README.md]: Oh I see now, sorry. How will this be displayed?

File: README.md [hunk 2]
```diff
@@ -1572,16 +1572,16 @@ sayHi();
...
 <details><summary><b>Answer</b></summary>
 <p>
 
 #### Answer: D
 
-With the `throw` statement, we can create custom errors. With this statement, you can throw exceptions. An exception can be a <b>string</b>, a <b>number</b>, a <b>boolean</b> or an <b>object</b>. In this case, our exception is the string `'Hello world'`.
+With the `throw` statement, we can create custom errors. With this statement, you can throw exceptions. An exception can be a <b>string</b>, a <b>number</b>, a <b>boolean</b> or an <b>object</b>. In this case, our exception is the string `'Hello world!'`.
 
-With the `catch` statement, we can specify what to do if an exception is thrown in the `try` block. An exception is thrown: the string `'Hello world'`. `e` is now equal to that string, which we log. This results in `'Oh an error: Hello world'`.
+With the `catch` statement, we can specify what to do if an exception is thrown in the `try` block. An exception is thrown: the string `'Hello world~'`. `e` is now equal to that string, which we log. This results in `'Oh an error! Hello world!'`.
```
Comment (lydiahallie) [inline README.md]: Typo `~`

File: README.md (hunk 1 above)
Comment (gabrielsroka) [reply README.md]: It won't be displayed. It's an anchor, the link to it is at the top of the page.

Markdown will also generate anchors, so you can use one of those if you like.

**Comment by gabrielsroka** [comment]
You can skip the link since you just  added the new one from the other pr, but the other typos have not yet been committed.
